### ISSUE TITLE
Is there a cli command for Boot firmware?

### ISSUE THREAD

**Issue by RokkumanX** [Arisotura/melonDS#1278]
Hi

This isn't an issue per se and not a deal breaker by any means but I can't seem to find any information whether you can straightly boot firmware from cli or not and it's not covered in the Howto/FAQ either.

When the emulator is open there is an option to boot firmware and it works perfectly fine (I only use melonDS for DSi firmware/nand which I only have 1 game on that I care for)

Is there a cli command for starting melonDS and booting straight to firmware?

Otherwise this might be better suited as a Feature Request.

Thanks in advance!

**Comment by nadiaholmquist** [comment]
There aren't currently any command line parameters, though adding them is planned.

**Comment by RokkumanX** [comment]
Thank you very much for the feedback, now I know it's planned at least.

Have a good one!

**Comment by RokkumanX** [comment]
I actually created a nice little circumvention to my dilemma, in my cli launcher I pointed to the game and unchecked the option Boot game directly in the melonDS GUI.

So when I launch the command I get straight into the firmware just as I want to.

If I launch the game (DSi) standalone with that option checked nothing happens and the emulator sorta softlocks/freezes, not that it really matters anyway (I know DSi emulation very experimental) because the specific game is already installed in my nand backup.

Cheers!
